.github/workflows/labeler-cache-retention.yml

```diff
@@ -0,0 +1,35 @@
+# Regularly restore the prediction models from cache to prevent cache eviction
+name: "Labeler: Cache Retention"
```

Comment:
This is a lot of boilerplate.  I imagine it's just xcopied from somewhere.  Could you either add readme or comments with details on where this is copied from and how we update it?

Reply:
Good call; thanks. I added some comments to the workflows and a markdown doc capturing the info.